src/overlay.jsx

```diff
@@ -134,6 +131,16 @@ const Overlay = React.createClass({
     if (!this.props.autoLockScrolling) this._allowScrolling();
```

Comment:
You removed the function call to this method.
So I think that we can remove `preventScrolling` and `allowScrolling`.
This component is also used by `LeftNav`, I have just checked, it should be fine.